src/app/code/community/Zendesk/Zendesk/Block/Adminhtml/Config/Buttons/Signup.php

```diff
@@ -61,23 +75,10 @@ public function getPostInfo()
             'magento_callback' => Mage::helper('adminhtml')->getUrl('adminhtml/zendesk/redirect', array('type' => 'settings', 'id' => 'zendesk')),
             'magento_locale' => Mage::getStoreConfig('general/locale/code'),
             'magento_timezone' => Mage::getStoreConfig('general/locale/timezone'),
-            'magento_api_url' => Mage::getUrl('zendesk/api', array('_store' => $stores[1]->getDefaultStore()->getCode()))
```

Comment:
Using `$stores[1]` here is an incorrect assumption since the indexes here are the database primary key. This will throw a fatal error if no core_store_group with primary key 1 exists.